gvg_mapper/GVGNode[] nodes
gvg_mapper/GVGEdgeMsg[] edges
---
bool success
